docs/pages/guide/01-start.md

```diff
@@ -1,6 +1,122 @@
+
+<img src="assets/images/Yavin_logo.png" width="200"/>
+
+Yavin is a framework for rapidly building custom data applications that
+offers both a UI and an API. In addition, Yavin can be deployed as a
+standalone Business Intelligence tool. It is <span class="c1">scalable
+and secure</span> and it’s <span
+class="c1">intuitive</span> visualization interface allows for <span
+class="c1">fast drill-downs</span> of very large data sets through
+simple interface and visualization blocks. With Yavin, you can easily
+and intuitively create <span class="c1">reports</span>, which focus on
+arbitrary drill downs for a single visualization, or <span
+class="c1">dashboards</span>, which are collections of visualizations,
+for <span class="c1">rapidly distilling information</span><span
+class="c0">. </span>
+
+<span class="c0"></span>
+
+Disclaimer : For the sample demos that come with Yavin, we are using the
+“<span
+class="c17"><a href="https://www.google.com/url?q=https://www.kaggle.com/shivamb/netflix-shows&amp;sa=D&amp;ust=1606182367630000&amp;usg=AOvVaw0vjjwg_hSHcq80D3AjOpp1" class="c3">Netflix Movie and TV Shows</a></span>”
+data set that is sourced from <span
+class="c17"><a href="https://www.google.com/url?q=https://www.kaggle.com/&amp;sa=D&amp;ust=1606182367630000&amp;usg=AOvVaw2MS0pMaHmAABG0JmQABGOV" class="c3">Kaggle</a></span><span
+class="c0"> (Timestamped 11/2020). The use of this data is only for
+demonstrating the ease of onboarding any dataset to Yavin. And has no
+correlation with Yavin as a product or an Open Source Software.</span>
+
+<span class="c0"></span>
+
+<span class="c0">Yavin is a combination of three major modules.  They
+are:</span>
+
+- <span class="c0 c15">Denali (The design layer) </span>
+- <span class="c0 c15">Elide (The Web Service layer) </span>
+- <span class="c0 c15">Navi (The UI layer)</span>
+
+<span class="c0"></span>
+
+<span class="c0">Here is a short video of Yavin “in action” : </span>
+
+<img style="border:2px solid black;" src="assets/images/Get_to_know_yavin.gif" width="800" />
+<span class="c18"> Yavin simplicity “in action”</span>
+
+<span class="c0"></span>
+
+## <span class="c5">Building with Legos ® & Duplos ®</span>
+
+<span class="c0">In the software industry, when discovering patterns, we
+often build abstractions and libraries to provide reusable software. The
+D3.JS charting library, Spring Security, and the Flask web framework are
+all great examples. We often use the metaphor building with Legos ® to
+illustrate the process of building software with many libraries and
+frameworks. While this provides a significant productivity gain,
+especially when compared to writing everything from scratch, it still is
+costly to build applications. When building applications, it takes time
+to design a system, wire together libraries, test your logic, handle
+security concerns, develop a great user interface, build features, and
+battle test your system. With Carbon, we aim to package fully featured
```

Comment:
Not sure if we should use the words legos and duplos here.

I don't think this adequately explains a duplo.  Firstly, not all duplos are "experiences".  The APIs are considered duplos - but have no UI component.

I think a Duplo is a reusable block of functionality that is conceptually larger than a library that enables larger patterns of software reuse.  There is a tradeoff here though.  Duplos are not as flexible as writing code from scratch.  So you gain in functionality but you lose some ability to customize.  This distinction needs to be called out.